### PR TITLE
feat(url): Add SeriesInstanceUIDs wado query param

### PR DESCRIPTION
**Pull request by igoroctaviano**
Add a way to retrieve a subset of series from a study

### Context

This will only open the viewer with one series (one displaySet) loaded, and no
queries made for any other series.

Sometimes you need to only retrieve a subset of series in a study, you can do
that by providing study level wado query parameters in the URL such as
SeriesInstanceUIDs. For example:

```js
http://localhost:3000/viewer?StudyInstanceUIDs=1.3.6.1.4.1.25403.345050719074.3824.20170125113417.1&SeriesInstanceUIDs=1.3.6.1.4.1.25403.345050719074.3824.20170125113545.4,1.3.6.1.4.1.25403.345050719074.3824.20170125113545.5
```

### Changes & Results

- Add new query param to filter series
- Update retrieve study metadata services

### Testing

Go to:
```js
http://localhost:3000/viewer?StudyInstanceUIDs=1.3.6.1.4.1.25403.345050719074.3824.20170125095438.5
```
All series are retrieved.

Go to:
```js
http://localhost:3000/viewer?StudyInstanceUIDs=1.3.6.1.4.1.25403.345050719074.3824.20170125095438.5&SeriesInstanceUIDs=1.3.6.1.4.1.25403.345050719074.3824.20170125095438.11,1.3.6.1.4.1.25403.345050719074.3824.20170125095449.8
```
Only the second and third series are retrieved.

#### PR

- [x] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [x] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates

- [x] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [x] OS: macOS 14.0 (23A344)]
- [x] Node version: 16.20
- [x] Browser: Chrome 118.0.5993.88

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by sedghi** [comment]
can we merge the other PR by @rodrigobasilio2022 since he has been working on it for a long time (multi server) and I feel bad if I merge this and he has to rebase and resolves conflicts

**Comment by igoroctaviano** [comment]
> can we merge the other PR first by @rodrigobasilio2022 since he has been working on it for a long time (multi server) and I feel bad if I merge this and he has to rebase and resolves conflicts

Np! thanks!

**Comment by igoroctaviano** [comment]
@sedghi now that the multi server implementation is going to change, can we merge this one?

**Comment by sedghi** [comment]
@igoroctaviano Can you update the testing notes? how can i test it

**Comment by igoroctaviano** [comment]
> @igoroctaviano Can you update the testing notes? how can i test it

Added! thx.

File: extensions/default/src/DicomWebDataSource/utils/retrieveMetadataFiltered.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        StudyInstanceUID,
+        enableStudyLazyLoad,
+        seriesSpecificFilters,
+        sortCriteria,
+        sortFunction
+      );
+    });
+
+    Promise.all(promises).then(results => {
+      const result = { preLoadData: [], promises: [] };
```
Comment (sedghi) [inline extensions/default/src/DicomWebDataSource/utils/retrieveMetadataFiltered.js:44]: aggregatedResult?

**Review by sedghi — COMMENTED** [review]
Looks good, just a small comment

File: extensions/default/src/DicomWebDataSource/utils/retrieveMetadataFiltered.js (hunk 1 above)
Comment (igoroctaviano) [reply extensions/default/src/DicomWebDataSource/utils/retrieveMetadataFiltered.js:44]: What did you mean?

Comment (sedghi) [reply extensions/default/src/DicomWebDataSource/utils/retrieveMetadataFiltered.js:44]: rename request for the result

Comment (igoroctaviano) [reply extensions/default/src/DicomWebDataSource/utils/retrieveMetadataFiltered.js]: Done, thx!

**Review by sedghi — APPROVED** [review]

**Pull request merged**